### PR TITLE
update to absolute references to webdriver-io-core

### PR DESCRIPTION
**Pull request by FelixZilber**
## Proposed changes
update references to webdriver-io-core to absolute

## Types of changes
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments
fix [https://github.com/webdriverio/webdriverio/issues/3639](url)

### CODE REVIEW

**Comment by CrispusDH** [comment]
It nice to see that you raised up this corner case and fix it. Good job!

File: packages/webdriverio/webdriverio.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/// <reference types="./webdriverio-core"/>
+/// <reference types="webdriverio/webdriverio-core"/>
```
Comment (mgrybyk) [inline packages/webdriverio/webdriverio.d.ts]: this one is good

File: packages/wdio-sync/webdriverio.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/// <reference types="./webdriverio-core"/>
+/// <reference types="wdio/sync/webdriverio-core"/>
```
Comment (mgrybyk) [inline packages/wdio-sync/webdriverio.d.ts]: this one was good but not now :)

we should have @ here
@wdio/sync/webdriverio-core

Comment (FelixZilber) [reply packages/wdio-sync/webdriverio.d.ts]: 🤦‍♂️

**Comment by mgrybyk** [comment]
@christian-bromann ready to merge

**Pull request merged**